PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,11 +17,24 @@ open. -->
 
 #### Release Notes
 
-<!-- Write the release notes for this release below. See
-https://github.com/sympy/sympy/wiki/Writing-Release-Notes for more information
-on how to write release notes. The bot will check your release notes
-automatically to see if they are formatted correctly. -->
+<!-- Write the release notes for this release below between the BEGIN and END
+statements. The basic format is a bulleted list with the name of the subpackage
+and the release note for this PR. For example:
+
+* solvers
+  * Added a new solver for logarithmic equations.
+
+* functions
+  * Fixed a bug with log of integers.
+
+or if no release note(s) should be included use:
+
+NO ENTRY
```

Comment:
IMO we should omit the `NO ENTRY` instructions. Too many people already skip writing release notes with `NO ENTRY` when they really should include one.

Reply:
The explanation of how to not include an entry needs to be somewhere. I now know it automatically but I used to have to check this instruction every time. The particular invocation "NO ENTRY" doesn't seem intuitive to me.

Reply:
I always remember it, but I guess I also invented it :) Suggestions on a better way to write it are welcome.

It is still described on the wiki page which is linked to in the text here.